lib/ansible/module_utils/basic.py

```diff
@@ -1612,6 +1613,9 @@ def _check_arguments(self, check_invalid_arguments, spec=None, param=None, legal
             elif k == '_ansible_shell_executable' and v:
                 self._shell = v
 
+            elif k == '_ansible_command_warnings' and v:
+                self._command_warn = v
+
```

Comment:
I think that handling of _command_warn should be done in the command module since it is only useful for command.  Adding it as an undocumented option seems like it would work there.

Reply:
as a dupe to the existing 'warn' option that this sets the default for?